litellm/utils.py

```diff
@@ -11129,7 +11129,7 @@ def handle_clarifai_completion_chunk(self, chunk):
             completion_tokens = len(encoding.encode(text))
             return {
                 "text": text,
-                "is_finished": True,
+                "is_finished": False,
```

Comment:
when is this ever finished? @mogith-pn 

if you fake streaming isn't the first chunk also the last one?